### ISSUE TITLE
When installing using pip install pysentimiento the analyzer.py and __init__.py are distinct than the ones in the repository

### ISSUE THREAD

**Issue by difemaro** [pysentimiento/pysentimiento#34]
**Describe the bug**
When installing using pip install pysentimiento the analyzer.py and __init__.py are distinct than the ones in the repository. It is installing 0.2.5 instead of 0.4.2

**To Reproduce**
Using Python 3.10.6, I tried installing using **pip install pysentimiento** however the analyzer.py and __init__.py files are different from the ones in the github repository.

To make the code work, I have to download the files from the github repository and replace them.

**Expected behavior**
How can we install from the github repository directly?, I tried but it was not possible.

**Environment**
pip freeze:
absl-py==1.2.0
aiohttp==3.8.1
aiosignal==1.2.0
astunparse==1.6.3
async-timeout==4.0.2
attrs==22.1.0
Automat==20.2.0
cachetools==5.2.0
certifi==2022.6.15
cffi==1.15.1
charset-normalizer==2.1.1
click==8.1.3
cloudpickle==2.1.0
colorama==0.4.5
configparser==5.3.0
constantly==15.1.0
coverage==6.4.4
coveralls==3.3.1
cryptography==37.0.4
datasets==2.4.0
defusedxml==0.7.1
dill==0.3.5.1
docopt==0.6.2
emoji==2.0.0
exceptiongroup==1.0.0rc9
filelock==3.8.0
Flask==2.2.2
Flask-Cors==3.0.10
Flask-WTF==1.0.1
flatbuffers==1.12
frozenlist==1.3.1
fsspec==2022.8.2
future==0.18.2
gast==0.4.0
genson==1.2.2
google-auth==2.11.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.48.1
h5py==3.7.0
huggingface-hub==0.9.1
hyperlink==21.0.0
hypothesis==6.54.5
idna==3.3
incremental==21.3.0
iniconfig==1.1.1
itsdangerous==2.1.2
Jinja2==3.1.2
joblib==1.1.0
jsonschema==4.15.0
keras==2.9.0
Keras-Preprocessing==1.1.2
libclang==14.0.6
Markdown==3.4.1
MarkupSafe==2.1.1
mock==4.0.3
multidict==6.0.2
multiprocess==0.70.13
nltk==3.7
numpy==1.23.2
oauthlib==3.2.0
opt-einsum==3.3.0
packaging==21.3
pandas==1.4.4
pluggy==1.0.0
protobuf==3.19.4
py==1.11.0
pyarrow==9.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.21
PyJWT==2.4.0
pyOpenSSL==22.0.0
pyparsing==3.0.9
pyrsistent==0.18.1
pysentimiento==0.2.5
pytest==7.1.3
pytest-cov==3.0.0
python-dateutil==2.8.2
pytz==2022.2.1
PyYAML==6.0
regex==2022.8.17
requests==2.28.1
requests-oauthlib==1.3.1
responses==0.18.0
rsa==4.9
scikit-learn==1.1.2
scipy==1.9.1
sentiment-analysis-spanish==0.0.25
simplejson==3.17.6
six==1.16.0
sklearn==0.0
sortedcontainers==2.4.0
tableauserverclient==0.19.0
tabpy==2.5.0
tensorboard==2.9.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.9.2
tensorflow-estimator==2.9.0
tensorflow-io-gcs-filesystem==0.26.0
termcolor==1.1.0
textblob==0.17.1
threadpoolctl==3.1.0
tokenizers==0.12.1
tomli==2.0.1
torch==1.12.1
tornado==6.2
tqdm==4.64.1
transformers==4.21.3
Twisted==22.4.0
twisted-iocpsupport==1.0.2
typing_extensions==4.3.0
urllib3==1.26.12
Werkzeug==2.2.2
wrapt==1.14.1
WTForms==3.0.1
xxhash==3.0.0
yarl==1.8.1
zope.interface==5.4.0

python --version
Python 3.10.6

**Additional context**
Once I have replaced the files with the ones in the repository, it worked.

**Comment by finiteautomata** [comment]
Hi @difemaro, it seems you have an old version of pysentimiento (0.2.5). Try installing the latest (4.2.0)

**Comment by difemaro** [comment]
Could you help me how to do it?, I am not python expert.

Diego

**Comment by finiteautomata** [comment]
```
pip install -U pysentimiento
```

should do it.

**Comment by difemaro** [comment]
Hi, I think the packages available for python 3.10 does not let me download the 0.4.2 release:

C:\Users\difem>pip install pysentimiento==0.4.2
ERROR: Ignored the following versions that require a different python version: 0.3.0 Requires-Python >=3.7,<3.10; 0.3.2 Requires-Python >=3.7,<3.10; 0.4.0 Requires-Python >=3.7,<3.10; 0.4.1 Requires-Python >=3.7,<3.10; 0.4.2 Requires-Python >=3.7,<3.10
ERROR: Could not find a version that satisfies the requirement pysentimiento==0.4.2 (from versions: 0.0.1.2, 0.1.0rc1, 0.1.1, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.2.5)
ERROR: No matching distribution found for pysentimiento==0.4.2

**Comment by finiteautomata** [comment]
You are right. I have to check pysentimiento is working in 3.10. I leave this open for further information.

In the meanwhile, if you can switch to 3.9, it should work.

**Comment by finiteautomata** [comment]
Hi @difemaro. Try `pysentimiento 0.5.2` which now has 3.10-support.
